nix/overlays/common-lib.nix

```diff
@@ -39,13 +39,24 @@ in {
       go;
 
     # Get the index-state and ghc version out of a cabal.project file.
+    #
+    # TODO: It would be better to re-use functions from haskell.nix like:
```

Comment:
Lets add a JIRA & tag it with "devops" and reference here.

Reply:
Created and added reference to https://input-output.atlassian.net/browse/ADP-2445